debian/changelog

```diff
@@ -1,3 +1,20 @@
+opx-nas-ndi-api (7.7.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
+  * Update: Use vlan id for pv bridge port get
+  * Update: Add API to set bridge port attributes
+  * Update: Add support for virtual rif attribute as part of RIF configuration
+  * Update: Add support for setting multicast lookup key in VLAN
+  * Update: Add APIs to clear statistics for bridge,bridge port and tunnel
```

Comment:
need a space between "bridge," and "bridge port". Also, add Oxford comma after "bridge port".